core/core-security/src/main/java/com/pgms/coresecurity/security/jwt/JwtTokenProvider.java

```diff
@@ -29,28 +30,26 @@
 import io.jsonwebtoken.security.Keys;
 
 @Component
-public class JwtUtils {
-	private static final Logger logger = LoggerFactory.getLogger(JwtUtils.class);
+public class JwtTokenProvider {
+	private static final Logger logger = LoggerFactory.getLogger(JwtTokenProvider.class);
 
 	@Value("${jwt.secret-key}")
 	private String secretKey;
 
 	@Value("${jwt.expiry-seconds}")
 	private int expirySeconds;
 
-	public String generateJwtToken(Authentication authentication) {
-		UserDetailsImpl userPrincipal = (UserDetailsImpl)authentication.getPrincipal();
-
+	public String generateJwtToken(UserDetailsImpl userDetails) {
```

Comment:
```suggestion
	public String generateAccessToken(UserDetailsImpl userDetails) {
```
으로 하는 것이 조금 더 명확해 보입니다!

둘을 함께 발급하는 메서드 이름을 generateJwtToken 으로 하는 것은 어떨까요??

Reply:
좋습니다 👍